src/saving/SaveList.cs

```diff
@@ -66,6 +69,7 @@ public class SaveList : ScrollContainer
     private CustomConfirmationDialog upgradeSaveDialog = null!;
     private CustomConfirmationDialog loadIncompatiblePrototypeDialog = null!;
     private ErrorDialog upgradeFailedDialog = null!;
+    private ErrorDialog errorSaveDeletionFailed = null!;
```

Comment:
While it's not entirely wrong to use `ErrorDialog`, the actual established practice for displaying errors for this kinds of operation is just to use our plain `CustomConfirmationDialog` for friendlier user interface. For more info, see #3947.

I know this is counter-intuitive but take it as an example of [Maslow's hammer](https://en.wiktionary.org/wiki/if_all_you_have_is_a_hammer,_everything_looks_like_a_nail), I guess.